### PR TITLE
added more options to run left right or straight

### PR DESCRIPTION
**Pull request by matthewmagee05**
Please review this, @casey-collab

### CODE REVIEW

File: english/new-movie/new-movie.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ Clues, clues, everywhere clues that don't connect!
 
 Dammit! No way you're going to miss this movie, it's been years you've been waiting for it. Time to run!
 
+Do you:
+
+[run left?](../super-man/save-the-word.md)
+
+[run right?](../suprise/suprise.md)
```
Comment (casey-collab) [inline english/new-movie/new-movie.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/new-movie/new-movie.md [hunk 2]
```diff
@@ -27,3 +27,11 @@ Clues, clues, everywhere clues that don't connect!
...
+
+[run straight?](../undending-story/undending-story.md)
```
Comment (casey-collab) [inline english/new-movie/new-movie.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by matthewmagee05** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cherry_blossom:

**Pull request merged**